samples/bluetooth/peripheral_hids_mouse/sample.yaml

```diff
@@ -21,3 +21,11 @@ tests:
     platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
       nrf5340dk_nrf5340_cpuapp_ns
     tags: bluetooth ci_build
+  sample.bluetooth.peripheral_hids_mouse.ble_rpc:
+    build_only: true
+    extra_configs:
+      - CONFIG_BT_RPC_STACK=y
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp
```

Comment:
is it possible to run nRF RPC with the ns target? If so, we can also add NS board variants to the test yaml

Reply:
At this moment It cannot be built for ns target. Seems like there are some additional dependencies that would need to be serialized to run this on non secure target